openedge-plugin/src/main/java/org/sonar/plugins/openedge/foundation/licence/SonarLintSideNameProvider.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * OpenEdge plugin for SonarQube
+ * Copyright (C) 2013-2016 Riverside Software
+ * contact AT riverside DASH software DOT fr
+ * 
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
+ */
+package org.sonar.plugins.openedge.foundation.licence;
+
+import org.sonarsource.api.sonarlint.SonarLintSide;
+
+/**
+ * Dummy implementation when analysis is executed in SonarLint
+ */
+@SonarLintSide
+public class SonarLintSideNameProvider implements IServerNameProvider {
+  public SonarLintSideNameProvider() {
+    System.out.println("Building SonarLintSideNameProvider");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS106)